scan-payment/src/main/java/com/getbouncer/scan/payment/ml/AlphabetDetect.kt

```diff
@@ -0,0 +1,111 @@
+package com.getbouncer.scan.payment.ml
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Rect
+import android.util.Log
+import android.util.Size
+import com.getbouncer.scan.framework.Loader
+import com.getbouncer.scan.framework.ResourceLoader
+import com.getbouncer.scan.framework.image.crop
+import com.getbouncer.scan.framework.image.hasOpenGl31
+import com.getbouncer.scan.framework.image.scale
+import com.getbouncer.scan.framework.image.size
+import com.getbouncer.scan.framework.image.toRGBByteBuffer
+import com.getbouncer.scan.framework.ml.TFLAnalyzerFactory
+import com.getbouncer.scan.framework.ml.TensorFlowLiteAnalyzer
+import com.getbouncer.scan.framework.util.scaleAndCenterWithin
+import com.getbouncer.scan.payment.R
+import java.io.FileNotFoundException
+import java.nio.ByteBuffer
+import org.tensorflow.lite.Interpreter
+
+private val TRAINED_IMAGE_SIZE = Size(48, 48)
+
+/** model returns whether or not there is a screen present */
+private const val NUM_CLASS = 27
+class AlphabetDetect private constructor(interpreter: Interpreter) :
+    TensorFlowLiteAnalyzer<AlphabetDetect.Input, ByteBuffer,
+            AlphabetDetect.Prediction,
+            Array<FloatArray>>(interpreter) {
+
+    companion object {
+        /**
+         * Calculate what portion of the full image should be cropped for screen detection.
+         */
+        fun calculateCrop(fullImage: Size): Rect =
+            Size(9, 16).scaleAndCenterWithin(fullImage) // Expects 16:9 aspect ratio
```

Comment:
This is a dangerous assumption, as it always assumes portrait orientation. We plan to support landscape orientation as well soon.

Reply:
It looks like this method might not be needed.